rca/settings/base.py

```diff
@@ -724,16 +724,16 @@
     "birdbath.checks.contrib.heroku.HerokuNotProductionCheck",
 ]
 BIRDBATH_PROCESSORS = [
-    "birdbath.processors.users.UserEmailAnonymiser",
-    "birdbath.processors.users.UserPasswordAnonymiser",
+    "rca.users.birdbath.StudentAccountAnonymiser",
```

Comment:
I found that Student User Accounts have a related Page. The pages also have data thats personal. This anonymises.

- Student User Account
- Student Page
- Related Student Collections
- Related Inline Records of a Student Page